src/components/Annotation.tsx

```diff
@@ -50,9 +50,7 @@ const Content = ({
       {description && (
         <Paragraph
           style={{
-            margin: `${suomifiDesignTokens.spacing.m} ${
-              suomifiDesignTokens.spacing.m
-            } 0 ${suomifiDesignTokens.spacing.m}`,
+            margin: `${suomifiDesignTokens.spacing.m} ${suomifiDesignTokens.spacing.m} 0 ${suomifiDesignTokens.spacing.m}`,
```

Comment:
This line looks long. There might be something wrong with my prettier settings. You could try running prettier check when testing to see if it says something about this.

Reply:
I tested to run `yarn validate` and `yarn format:code` but nothing was found/changed. So maybe there are no current rules for that.
This repo doesn't have the same rules as components-repo, at least not yet.